decidim-admin/lib/decidim/admin/test/filterable_examples.rb

```diff
@@ -1,13 +1,13 @@
-# frozen_string_literal: true
+# rubocop:disable Style/FrozenStringLiteralComment
```

Comment:
I'm curious, why do you need to remove the frozen literal? 🤔 🤯

Reply:
I _thought_ this could be the solution to the ever failing tests I have right now 😅 
I disabled the magic comment because I need to 'overwrite' the constant `FILTERABLE_CONCERN`. Made is a string and disabled the frozen literals so that it'd let me change it (sneaky and dirty, but I am really running out of ideas).

In short, what's happening is that in the same test suite (`decidim-elections`) that constant is set twice - once when we filter `votings` (hence set to the votings filterable concern) and when filtering `polling_officers` (set to the polling officers filterable concern).
The tests that gets executed first will pass, but the one that comes second will have the wrong concern set and fail.

Anyways, I _thought_ this would finally set me free from this PITA but the check still goes red 😭

Reply:
https://github.com/decidim/decidim/pull/7402 will solve the issue, after rebasing on that the magic comment will be reestablished